### PR TITLE
Add GTFS sensor docs

### PR DESCRIPTION
**Pull request by robbiet480**
This adds documentation for the GTFS sensor added in balloob/home-assistant#1623.

### CODE REVIEW

File: source/_components/sensor.gtfs.markdown [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+layout: page
+title: "Public Transit (GTFS)"
+description: "Instructions how to use public transit open data inHome Assistant."
```
Comment (GreenTurtwig) [inline source/_components/sensor.gtfs.markdown]: Typo: Instructions on how to use public transit open data in Home Assistant.

File: source/_components/sensor.gtfs.markdown [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+The `gtfs` sensor will give you the next departure time and associated data from your public transit station/stop. The data comes from your chosen public transit authority and is formatted as [General Transit Feed Specification](https://developers.google.com/transit/gtfs/) data, commonly known as GTFS.
+
+You need to find a valid GTFS data set, which you can usually find just by searching the internet. Most public transit authorities have GTFS available somewhere, as Google requires public transit authorities to provide the data if they wish to appear on Google Maps.
+
+Here are some examples:
+
+- [Bay Area Rapid Transit (BART)](http://www.bart.gov/schedules/developers/gtfs) - The light rail system for the San Francisco Bay Area.
+- [Metropolitan Transit Authority of New York City (MTA)](http://www.bart.gov/schedules/developers/gtfs) - Provides separate data feeds for subway, bus, LIRR and Metro-North of the greater New York City metropolitan region.
+- [GBRail.info](http://www.gbrail.info/) - Provides data feeds for most if not all rail companies in the Britain.
```
Comment (GreenTurtwig) [inline source/_components/sensor.gtfs.markdown]: Typo: Provides data feeds for most if not all rail companies in Britain.

File: source/_components/sensor.gtfs.markdown [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+To find your stop ID, open the `stops.txt` file inside the ZIP file/unzipped folder. The format of the ID is different for every transit agency but will be the first "column" (meaning the string before the first comma) in a row.
+
+The sensor attributes will contain all related information for the specific trip, such as agency information, origin and destination stop information, origin and destination stop time and the route information.
+
+Your mileage may vary depending on the transit agency used. Most agencies respect the GTFS format but some will do weird things like adding extra columns or using different data formatting. If you have any data specific issues, please report them to the [PyGTFS](https://github.com/jarondl/pygtfs) project, which is what the GTFS sensor uses to parse data.
+
+**Please note**: This is a _static_ data source. Currently, there is no GTFS Realtime support in this sensor due to issues surrounding parsing the protocol buffer format in Python 3. Once those issues have been fixed Realtime support will be added. Once added, the sensor will check for any delays and advisories and report them in sensor as needed.
```
Comment (GreenTurtwig) [inline source/_components/sensor.gtfs.markdown]: Typo: them in the sensor as needed.

**Comment by GreenTurtwig** [comment]
Read through it all and added comments for any typos/mistakes I found. Looks good. :-)

**Comment by MinchinWeb** [comment]
Since data needs to be downloaded, does it make sense to provide a list of links to sources on the bottom of the page? Or links on a separate page?

For example, Edmonton AB is at https://data.edmonton.ca/d/ckp7-ccr2?category=Transit&view_name=ETS-Bus-Schedule-GTFS-Data-Feed-Link-for-Developer

**Comment by balloob** [comment]
It will be impossible to link them all. Link a few big ones and link to the
page that collects them all

**Comment by robbiet480** [comment]
@MinchinWeb There's just soooo many links to provide. If anything, we should link to transitfeeds.com or  www.gtfs-data-exchange.com

**Comment by balloob** [comment]
@robbiet480 could you add the two links to transit feeds / data exchange? After that it looks good to go.

**Comment by MinchinWeb** [comment]
@robbiet480 In a dream world, it would be nice to just specify the name of your transit provider, and have Home Assistant download the appropriate file. But I realize that would be ... involved, especially if someone has to maintain a link list. Would the [Transit Feeds API](http://transitfeeds.com/api/) be able to provide the needed links/data?

**Comment by balloob** [comment]
Dream world would use gps of the home to find closest transit provider plus
station ;-)

**Comment by robbiet480** [comment]
@MinchinWeb So I'm hesitant to do that because of data updates. We would have to either track the last updated time from the API (in a different file, so now we are at 3 files total) or just download and load the data every single time, which doesn't really make sense because a lot of providers update once every 6 months. On the other hand, some providers update every 7 days.

@balloob That would be the dream, but the problem with that is when you are equidistant from two stations but they go in opposite directions. If I walk to one, I get to work in 15 minutes. Walk to the other (which may actually be a smidgen closer) and it's 25 minutes. So i'd still have to manually define the station (at least in my case). Also, there are 2 different transit providers available at my location, the local bus company and the light rail system.

**Pull request merged**